tests/components.test.js

```diff
@@ -53,11 +53,31 @@ it("Accepts an additional `className`", () => {
 
 it("Doesn't trigger `onChange` after mounting", () => {
   const handleChange = jest.fn();
-  render(<ColorPicker onChange={handleChange} />);
+  render(<ColorPicker color="#c62182" onChange={handleChange} />);
 
   expect(handleChange).not.toHaveReturned();
 });
 
+it("Doesn't trigger `onChange` after controlled rerender", () => {
+  const handleChange = jest.fn();
+  const { rerender } = render(<ColorPicker color="#c62182" onChange={handleChange} />);
+
+  rerender(<ColorPicker color="#c72282" onChange={handleChange} />);
+
+  expect(handleChange).not.toHaveReturned();
+});
+
+it("Doesn't trigger `onChange` when hex color is black", () => {
+  const handleChange = jest.fn();
+  const { container } = render(<ColorPicker color="#000" onChange={handleChange} />);
+  const hue = container.querySelector(".react-colorful__hue .interactive");
```

Comment:
Hmm. What's the goal of this test? Is there a case that other tests dont cover?

Reply:
Yes ofc
When you change hue for `#000` color picker fires `onChange`, because `hsv(120 0 0)` -> `#000` and `hsv(0 0 0)` is black to. I added test fro this case.

Reply:
Ow, I see. Nice one!
Should we rename the test maybe?
Something like 
```
... if changing the hue doesn't change the hex color (grayscale colors)
```

Reply:
Or
```
Doesn't call `onChange` when user changes a hue of a grayscale color`
```

Reply:
Very nice test btw. Thank you!

Reply:
I renamed test